commcare_connect/opportunity/tables.py

```diff
@@ -315,6 +316,25 @@ def render_revoke_suspension(self, record, value):
         return format_html('<a class="btn btn-success" href="{}?next={}">Revoke</a>', revoke_url, page_url)
 
 
+class CatchmentAreaTable(tables.Table):
+    id = columns.Column(verbose_name="Id")
```

Comment:
It might be nice to add the user phone number and name to these downloads (in addition to the username), because that will be much easier for users to identify the person than the username. It is ok to ignore those in the upload as we do in other downloads.

Reply:
Added